### PR TITLE
Created Input component

### PR DESCRIPTION
**Pull request by anushasunil**
Created input box , textarea, radio button and a checkbox 

https://user-images.githubusercontent.com/90134948/152412460-5a824552-3956-484c-a0ca-1f05bc0229d5.mp4

### CODE REVIEW

File: Stylesheets/Components/input.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    color: var(--color-primary);
+}
+
+.input-box input,
+.input-box textarea {
+    font-size: 1.2rem;
+    padding: 1rem;
+    border: 2px solid var(--color-primary);
+    border-radius: var(--container-border);
+    background-color: var(--color-tertiary-variant2);
```
Comment (ttahm3d) [inline Stylesheets/Components/input.css]: from what i have seen in video, this is not blending with the background of body. Try to change it to lighter shade of grey (lighter than body's background)

File: Stylesheets/utility.css [hunk 2]
```diff
@@ -34,6 +34,11 @@ body {
     margin: 0 auto;
 }
 
+.display-flex-column {
+    display: flex;
+    flex-direction: column;
+}
+
```
Comment (ttahm3d) [inline Stylesheets/utility.css:41]: Good job here on creating utility classes.

**Review by ttahm3d — COMMENTED** [review]
Good that you are using utility classes. Change colors slightly, to create a good contrast and visual hirearchy

**Review by vivekmehto — COMMENTED** [review]
All the variations look awesome.

File: Stylesheets/Components/input.css (hunk 1 above)
Comment (anushasunil) [reply Stylesheets/Components/input.css]: thankyou. I've rectified the color now
Please let me now if it is correct now

image

Comment (ttahm3d) [reply Stylesheets/Components/input.css]: Looks good now

**Review by ttahm3d — APPROVED** [review]

**Pull request merged**